### PR TITLE
feat: Tax based expense claim

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
- Provides a field for adding taxes in the expense claim doctype
Screenshot 2019-06-12 at 8 05 40 PM

- added a few UX changes in an expense claim
expense claim changes

- general ledger that gets created
Screenshot 2019-06-14 at 1 03 06 PM

### CODE REVIEW

**Comment by rohitwaghchaure** [comment]
@Mangesh-Khairnar Can you add the schreenshots of the GL entries created against the expense claim

**Comment by Mangesh-Khairnar** [comment]
@rohitwaghchaure added pics

File: erpnext/hr/doctype/expense_claim/expense_claim.js [hunk 1]
```diff
@@ -332,6 +358,50 @@ frappe.ui.form.on("Expense Claim Advance", {
...
 
+frappe.ui.form.on("Expense Taxes and Charges", {
+	account_head: function(frm, cdt, cdn) {
+		var child = locals[cdt][cdn];
+		if(child.account_head && !child.description && !child.rate) {
+			// set description from account head
+			child.description = child.account_head.split(' - ').slice(0, -1).join(' - ');
+
+			// set the tax rate from account head
+			frappe.db.get_value("Account", child.account_head, "tax_rate").then((r) => {
```
Comment (rohitwaghchaure) [inline erpnext/hr/doctype/expense_claim/expense_claim.js]: No need of this code, use fetch from property to fetch rate from the account master

Comment (Mangesh-Khairnar) [reply erpnext/hr/doctype/expense_claim/expense_claim.js]: added changes

**Review by rohitwaghchaure — APPROVED** [review]

**Comment by vcavel** [comment]
Hi guys, do you know in which final version this will be ?

Thanks for the good job, we could really use this !

**Pull request merged**